lib/__tests__/findUndefinedIdentifiers-test.js

```diff
@@ -73,11 +73,21 @@ it('knows about jsx', () => {
   `,
       ),
     ),
-  ).toEqual(new Set([
-    'bar',
-    'React', // Implicit dependency
-    'FooBar',
-  ]));
+  ).toEqual(
+    new Set([
+      'bar',
+      'React', // Implicit dependency
+      'FooBar',
+    ]),
+  );
+});
+
+it('ignores lowercase jsx element identifiers', () => {
+  expect(findUndefinedIdentifiers(parse("export default <input value='foo' />;"))).toEqual(
+    new Set([
+      'React', // Implicit dependency
+    ]),
+  );
```

Comment:
This is the only significant change to this file (the rest is a result of running prettier on the content).